src/universal/components/InputField/InputField.js

```diff
@@ -1,57 +1,59 @@
 import React, {PropTypes} from 'react';
-import look, {StyleSheet} from 'react-look';
-import {Field as ReduxFormField} from 'redux-form';
-import appTheme from 'universal/styles/theme';
+import withStyles from 'universal/styles/withStyles';
+import {css} from 'aphrodite';
+import appTheme from 'universal/styles/theme/appTheme';
 import IconButton from 'universal/components/IconButton/IconButton';
 
-const combineStyles = StyleSheet.combineStyles;
-const fieldLightGray = appTheme.palette.dark50l;
-
-let styles = {};
-
-const renderField = (field) => {
+const InputField = (props) => {
   const {
+    autoFocus,
     hasErrorText,
     hasHelpText,
     helpText,
     input,
     isWider,
-    theme,
+    theme: inputTheme,
```

Comment:
ah, I'll change this to `colorPalette`.
`defaultProps` are ok in this case, but they aren't passed to the `withStyles` HOC, which means if our stylesThunk depends on the `colorPalette`, it'll be undefined. For that reason, I try to avoid defaultProps & explicitly pass that in, but I don't see a single case where anything using `InputField` passes in that value.